apps/ledger-live-mobile/src/families/elrond/components/Flows/Delegate/types.ts

```diff
@@ -20,6 +20,7 @@ export type ElrondDelegationFlowParamList = {
     account: ElrondAccount;
     transaction?: Transaction;
     source?: RouteProp<ParamListBase, ScreenName>;
+    trackingFirstStep?: boolean;
```

Comment:
```suggestion
    isFirstStep?: boolean;
```
or `trackAsFirstStep`